Submitted By: Wolfgang Arendt <wolare@gmx.de>
Date: 2003-09-26
Initial Package Version: 2.9b
Origin: None.
Description: Allow at-2.9b to compile. Check any sunsite mirror for the at daemon.
diff -ur at.orig/at.c at/at.c
--- at.orig/at.c	1995-10-30 14:41:56.000000000 -0600
+++ at/at.c	2003-09-27 00:57:17.000000000 -0500
@@ -44,7 +44,6 @@
 /* Local headers */
 
 #include "at.h"
-#include "panic.h"
 #include "parsetime.h"
 #include "perm.h"
 
diff -ur at.orig/panic.c at/panic.c
--- at.orig/panic.c	1995-10-30 14:41:57.000000000 -0600
+++ at/panic.c	2003-09-27 00:57:17.000000000 -0500
@@ -26,7 +26,6 @@
 
 /* Local headers */
 
-#include "panic.h"
 #include "at.h"
 
 /* File scope variables */
diff -ur at.orig/parsetime.c at/parsetime.c
--- at.orig/parsetime.c	1995-10-30 14:41:57.000000000 -0600
+++ at/parsetime.c	2003-09-27 00:57:17.000000000 -0500
@@ -43,7 +43,6 @@
 /* Local headers */
 
 #include "at.h"
-#include "panic.h"
 
 /* Structures and unions */
 
